tests/transforms/test_video_utils.py

```diff
@@ -39,39 +42,136 @@ def test_video_downsample(
     assert np.array_equal(downsampled_video, expected)
 
 
-@pytest.mark.parametrize("video_size", [((16, 32)),
-                                        ((16, 16))])
-def test_webm_conversion(video_size, tmp_path):
+def cmp_videos(encoded_video_path: str, expected_video: np.ndarray):
```

Comment:
Nit: `compare` vs `cmp` is much more readable

Reply:
Changed `cmp_videos` to `compare_videos`.